server/views/pages/recall/check-sentences.njk

```diff
@@ -104,7 +104,7 @@
 
 {%  macro sentenceCards(groups) %}
     {% for group in groups %}
-    {%  if group.hasEligibleSentences  %}
+{#    {%  if group.hasEligibleSentences  %}#}
```

Comment:
Is this change intentional? Should the check just be removed? Or at least something to explain why its commented out.